### ISSUE TITLE
Deprecate metrics debugger

### ISSUE THREAD

**Issue by benshayden** [catapult-project/catapult#2391]
Currently, we have both the metrics side panel and the metrics debugger.
The metrics side panel uses the same widgets as results2.html, in the context of the trace viewer.
The metrics debugger displays the raw json of metrics, but can load trace files in a more general way than trace viewer.

Let's deprecate the metrics debugger in favor of the side panel as a proving ground for results2.html.

Maybe we can retain loading trace files using a file picker, but as a separate issue?

@nedn

**Comment by nedn** [comment]
Loading trace files using a file picker is a critical feature to keep for people to do easy debugging. And since a few people already familiar with that metric debugger link, a solution here is to just replace the map_results (https://github.com/catapult-project/catapult/blob/master/tracing/tracing/ui/metrics_debugger_app.html#L34) of metric debugger with the <value-set-view>.

**Comment by benshayden** [comment]
This should still happen, but it's blocked on adding an Open button to trace file viewer, which is tied up with #1933 .
